tests/std/tests/Dev11_0235721_async_and_packaged_task/test.cpp

```diff
@@ -295,6 +295,8 @@ template <typename T, typename... Args>
 constexpr bool is_constructible_with_trailing_empty_brace =
     is_constructible_with_trailing_empty_brace_impl<void, T, Args...>;
 
+STATIC_ASSERT(is_constructible_with_trailing_empty_brace<pair<double*, int>, double*>); // verify a true case

```

Comment:
Me: "We don't need tests for such simple test machinery - you can remove these."
Stephan: Adds a test for the test machinery.

Reply:
It's partially a general principle about being methodical with testing, but also it's paranoia about compiler bugs. One never knows whether a compiler today or tomorrow will have trouble with this brace SFINAE and begin reporting `false` all the time. On a related note, I found several cases when working on Standard Library Modules where very simple tests that "obviously" passed for years, were just special enough to reveal bugs when the compiler began working totally differently.

Reply:
We had a wonderful discussion in the LWG telecon this morning about how much easier library development would be if it weren't for compilers.